Author: IOhannes m zm√lnig
Description: use system installation of portaudio/portmidi
 rather than Pd's own copy
--- puredata.orig/Makefile.am
+++ puredata/Makefile.am
@@ -12,14 +12,6 @@
 EXTRA_SUBDIRS += asio
 endif
 
-if PORTAUDIO
-EXTRA_SUBDIRS += portaudio
-endif
-
-if PORTMIDI
-EXTRA_SUBDIRS += portmidi
-endif
-
 # files that are included but not built
 EXTRA_DIST = LICENSE.txt README.txt INSTALL.txt
 
@@ -27,7 +19,7 @@
 SUBDIRS = $(EXTRA_SUBDIRS) src man tcl po extra
 
 # subdirs that are always included in the dist, etc.
-DIST_SUBDIRS = asio extra man po portaudio portmidi src tcl
+DIST_SUBDIRS = asio extra man po src tcl
 
 BUILT_SOURCES = 
 if MSGFMT
--- puredata.orig/configure.ac
+++ puredata/configure.ac
@@ -5,11 +5,6 @@
 AM_INIT_AUTOMAKE
 AC_CONFIG_MACRO_DIR([m4/generated])
 
-if test -d "$srcdir/portaudio"; then
-  AC_CONFIG_SUBDIRS([portaudio])
-fi
-
-
 AC_CANONICAL_HOST
 
 # you will need to install iPhone XCode on Mac OS X to use this:
@@ -251,7 +246,6 @@
 AC_CONFIG_FILES([Makefile
                 asio/Makefile
                 man/Makefile
-                portmidi/Makefile
                 tcl/Makefile
                 po/Makefile
                 src/Makefile
--- puredata.orig/src/Makefile.am
+++ puredata/src/Makefile.am
@@ -85,8 +85,8 @@
 endif
 
 if PORTAUDIO
-pd_CFLAGS += -DUSEAPI_PORTAUDIO  -I$(top_srcdir)/portaudio/include
-pd_LDADD += $(top_builddir)/portaudio/lib/libportaudio.la
+pd_CFLAGS += -DUSEAPI_PORTAUDIO
+pd_LDADD += -lportaudio
 pd_SOURCES += s_audio_pa.c s_audio_paring.c
 endif
 
@@ -96,8 +96,7 @@
 endif
 
 if PORTMIDI
-INCLUDES += -I$(top_srcdir)/portmidi/pm_common -I$(top_srcdir)/portmidi/porttime
-pd_LDADD += $(top_builddir)/portmidi/libportmidi.la
+pd_LDADD += -lportmidi -lporttime
 pd_SOURCES += s_midi_pm.c
 endif
 
